child_compassion/models/child_lifecycle_event.py

```diff
@@ -196,3 +211,15 @@ class ChildLifecycleEvent(models.Model):
         ('West Nile Virus', 'West Nile Virus'),
         ('Yellow Fever', 'Yellow Fever'),
     ], readonly=True)
+
+    @api.model
+    def process_commkit(self, commkit_data):
+        lifecycle_mapping = mapping.new_onramp_mapping(
+            self._name,
+            self.env,
+            'new_child_lifecyle')
+
+        vals = lifecycle_mapping.get_vals_from_connect(commkit_data)
+        lifecycle = self.with_context(no_comm_kit=True).create(vals)
```

Comment:
context no_commkit is not useful, because the `create` method of object `child lifecycle` does not create any message (the create method is not overriden).